src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -518,6 +517,30 @@ public void assertContainsIgnoringCase(AssertionInfo info, CharSequence actual,
       throw failures.failure(info, shouldContainIgnoringCase(actual, sequence));
   }
 
+
+  /**
+   * Verifies that the given {@code CharSequence} contains the given strings, ignoring whitespaces.
+   *
+   * @param info contains information about the assertion.
+   * @param actual the actual {@code CharSequence}.
+   * @param values the values to look for.
+   * @throws NullPointerException if the given sequence is {@code null}.
+   * @throws IllegalArgumentException if the given values is empty.
+   * @throws AssertionError if the given {@code CharSequence} is {@code null}.
+   * @throws AssertionError if the actual {@code CharSequence} does not contain the given sequence.
+   */
+  public void assertContainsIgnoringWhitespace(AssertionInfo info, CharSequence actual, CharSequence... values) {
+    doCommonCheckForCharSequence(info, actual, values);
+    String actualWithoutWhitespace = removeAllWhitespaces(actual);
+    Set<CharSequence> notFound = stream(values).filter(value -> !stringContains(actualWithoutWhitespace, removeAllWhitespaces(value)))
+      .collect(toCollection(LinkedHashSet::new));
+    if (notFound.isEmpty()) return;
+    if (notFound.size() == 1 && values.length == 1) {
```

Comment:
I dont think that `notFound.size() == 1` check is really needed. Because if second one is true then first one is also true, and we only need second one for choosing the right signature to call

Reply:
I think you are right that the first check ist not needed. Actually it is just a "copy&paste&modify" error of 
[Strings.assertContains(...)](https://github.com/assertj/assertj-core/blob/4459953d98c49e8f83c211ba7e00bf080d78ea89/src/main/java/org/assertj/core/internal/Strings.java#L463). I can change it here or leave it as is....